platform-tests/src/platform/availability/monitor/monitor.go

```diff
@@ -197,19 +200,26 @@ func (m *Monitor) Stop() {
 	}
 }
 
+func (m *Monitor) taskRateToDuration() time.Duration {
+	nanosecondsInASecond := time.Second.Nanoseconds()
+	return time.Duration(nanosecondsInASecond / m.taskRatePerSecond)
```

Comment:
I might just be having trouble mentally parsing this because it's morning, is this the same as just:

```
return time.Second / m.taskRatePerSecond
```

Reply:
Kind of. `time.Second` is of type `time.Duration`, so you can't do maths with it. I suppose I could have done a type cast:

```
return int64(time.Second) / m.taskRatePerSecond
```

Reply:
I've changed it to:

```
return time.Second / time.Duration(m.taskRatePerSecond)
```

for readability